data/json/mapgen/industrial/slaughterhouse.json

```
@@ -0,0 +1,238 @@
+[
+    {
+      "type": "item_group",
+      "id": "slaughter_raw_meat",
+      "items": [
+        { "item": "meat", "prob": 8, "charges": [ 1, 10 ] },
+        { "item": "lung", "prob": 1, "charges": [ 1, 20 ] },
+        { "item": "kidney", "prob": 1, "charges": [ 1, 20 ] },
+        { "item": "liver", "prob": 1, "charges": [ 1, 10 ] },
+        { "item": "brain", "prob": 1, "charges": [ 1, 10 ] },
+        { "item": "sweetbread", "prob": 2, "charges": [ 1, 10 ] },
+        { "item": "fat", "prob": 2, "charges": [ 5, 30 ] }
+      ]
+    },
+    {
+      "type": "item_group",
+      "id": "slaughter_tools",
+      "items": [
+        [ "knife_butcher", 30 ],
+        [ "tongs", 10 ],
+        [ "hacksaw", 10 ]
+      ]
+    },
+    {
+      "type": "item_group",
+      "id": "slaughter_clothes",
+      "items": [
+        [ "apron_leather", 30 ],
+        [ "jumpsuit", 30 ],
+        [ "gloves_rubber", 10 ],
+        [ "boots_rubber", 10 ],
+        [ "gloves_cut_resistant", 10 ]
+      ]
+    },
+    {
+      "id": "f_hanging_meathook",
+      "type": "furniture",
+      "name": "hanging meathook",
+      "symbol": "g",
+      "required_str": -1,
+      "move_cost_mod": 2,
+      "color": "light_gray",
+      "deconstruct": { "items": [ { "item": "grip_hook", "count": 1 }, { "item": "chain", "count": 1 } ] },
+      "flags": [ "TRANSPARENT", "SLAUGHTER_EQ" ]
```

Furniture and itemgroups that are added should go into separate files, such as furniture.json or one of the itemgroups files.
Also, 
```suggestion
      "flags": [ "TRANSPARENT", "BUTCHER_EQ" ]
```